Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -15,59 +15,47 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     public static class DispatcherHelper
     {
         /// <summary>
-        /// Executes the given function asynchronously on UI thread of the main view.
+        /// Executes the given function asynchronously on given view's UI thread. Default view is the main view.
         /// </summary>
-        /// <typeparam name="T">Returned data type of the function</typeparam>
         /// <param name="function">Asynchronous function to be executed asynchronously on UI thread</param>
```

Comment:
This is no longer true

Reply:
Good catch! I've improved the XML comments in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3119/commits/7dcce8a6e1561eb47c2c453ca4b0a821bcf0f472.